src/include/VM.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <DurianObject.h>
+
+#ifndef DURIAN_VM_H
+#define DURIAN_VM_H
+
+
+class VM {
+    public:
+        VM(unsigned char *);
+        ~VM();
+        int run();
+    private:
+        int pc;                  // Program Counter
+        int sp;                  // Stack Pointer
+        unsigned char *code;     // Pointer to bytecode
```

Comment:
You might want to consider [`std::vector`](http://en.cppreference.com/w/cpp/container/vector). In particular it would allow you to use an iterator for the program counter (c.f. [`.cbegin()`](http://en.cppreference.com/w/cpp/container/vector/begin)) which is a bit safer.